Makefile

```diff
@@ -34,9 +34,7 @@ githubRepoLink := https://github.com/Eppo-exp/sdk-test-data.git
 test-data: 
 	rm -rf $(testDataDir)
 	mkdir -p $(tempDir)
-	cd ${tempDir} \
-	    && git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} \
-	    && rm -rf RepoName/.git/
+	git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} ${gitDataDir}
```

Comment:
was hitting weird issues so modeled this off our JS SDK and things started working